census/templates/index.html

```diff
@@ -22,23 +20,47 @@
             </div>
 
             <div class="grid-col-8">
-                <div id="no-events-alert" class="usa-alert usa-alert--info display-none">
-                    <div class="usa-alert__body">
-                        <h3 class="usa-alert__heading">No events found for selected date</h3>
-                        <p class="usa-alert__text"></p>
+                {% if not request.has_events %}
+                    <div id="no-events-alert" class="usa-alert usa-alert--info">
+                        <div class="usa-alert__body">
+                            <h3 class="usa-alert__heading">No events found for selected date</h3>
+                            <p class="usa-alert__text"></p>
+                        </div>
+                    </div>
+                {% endif %}
+                {% if request.has_events %}
+                    <div id="event-list" class="usa-accordion usa-accordion--bordered">
+                        {% for event_data in request.events %}
+                            {# Each event data is a tuple with the 0th element  #}
+                            {# as the day and the 1st element as a list  #}
+                            {# of events  #}
+                            <label class="usa-label font-ui-sm text-bold" for="title">{{ event_data.0 }}</label>
+                            {% for event in event_data.1 %}
+                                <h2 class="usa-accordion__heading">
+                                    <button class="usa-accordion__button"
+                                            aria-expanded="true"
+                                            aria-controls="a{{ event.id }}">
+                                        <span>{{ event.start_time }} - {{ event.end_time }}</span>
+                                        <span class="text-normal">
+                                            {{ event.title }}
+                                        </span>
+                                    </button>
+                                </h2>
+                                <div id="a{{ event.id }}" class="usa-accordion__content usa-prose">
+                                    <p>{{ event.description }}</p>
+                                </div>
+                            {% endfor %}
+                        {% endfor %}
+                        {# This section is populated dynamically by events triggered#}
+                        {# in datepicker.js#}
                     </div>
-                </div>
-                <div id="event-list" class="usa-accordion usa-accordion--bordered">
-                    {# This section is populated dynamically by events triggered#}
-                    {# in datepicker.js#}
-                </div>
+                {% endif %}
             </div>
-            <div class="grid-col-4">
+
+            <div class="grid-col-4 desktop:display-block mobile: display-none">
```

Comment:
As the calendar causes issues on mobile screens, I have disabled the datepicker for mobile for now. Should we have a different style of datepicker for mobile ui? Or maybe a button to toggle the calendar view on/off?

Reply:
I think we can just use a regular date select for mobile.